src/Verse.ts

```diff
@@ -7,8 +7,7 @@ export default abstract class Verse {
 		private bookUrl: string,
 		private book: string,
 		private chapter: number,
-		private verse: number | undefined,
-		private verseEnd: number | undefined
+		private verses: string
```

Comment:
We cannot store verses as string entered by user - it breaks with en-dash support and other chapter separators like `,` and `.`. I suggest using list of objects like:
```ts
{
    start: number,
    end: number | undifined
}
```